web/src/elements/Interface.ts

```diff
@@ -4,14 +4,13 @@ import { AKElement } from "#elements/Base";
 import { BrandingContextController } from "#elements/controllers/BrandContextController";
 import { ConfigContextController } from "#elements/controllers/ConfigContextController";
 import { ModalOrchestrationController } from "#elements/controllers/ModalOrchestrationController";
-import { WithAuthentikConfig } from "#elements/mixins/config";
 
 import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
 /**
  * The base interface element for the application.
  */
-export abstract class Interface extends WithAuthentikConfig(AKElement) {
+export abstract class Interface extends AKElement {
```

Comment:
This is now handled more specifically by the element which consumes the context.